npm/ng-packs/packages/theme-shared/extensions/src/lib/components/extensible-table/extensible-table.component.ts

```diff
@@ -150,7 +150,7 @@ export class ExtensibleTableComponent<R = any> implements OnChanges {
 
   ngOnChanges({ data }: SimpleChanges) {
     if (!data?.currentValue) return;
-
+    this.list.totalCount = this.recordsTotal
```

Comment:
it seems code smells

Reply:
2 times sending request when the record created or deleted. We need to find an alternative solution

For now I found this

```ts
//extensible-table.component.ts

ngOnChanges({ data }: SimpleChanges) {
  if (!data?.currentValue) return;

  if (data.currentValue.length < 1) {
    this.list.totalCount = this.recordsTotal
  }
  ...
}
```

![image](https://github.com/abpframework/abp/assets/49063256/ddac55b5-396e-4ccf-9e32-9351f5bf92d7)